proposals/0000-local-do.rst

```diff
@@ -1,5 +1,5 @@
-Local do
-========
+Qualified do
```

Comment:
I kind of liked `LocalDo`. I don't mind the change either, but I'm not sure what's it's motivated by.

Reply:
Either name works for me. I'm changing it because QualifiedDo rang more appropriate with the new syntax. I think @mboes also expressed along this lines. Anyway, its up to you.

Reply:
I think `-XQualifiedDo` is more expressive about what the extension does, and more in keeping with existing nomenclature (qualified imports, qualified names, etc).